### DISCUSSION TITLE
I can't batch select and edit when using "Folder" mode. Can you add all the functions to make it the same as in "Photo" mode?/使用“文件夹”模式时无法批量选择，编辑。能否添加全部的功能，使其与“照片”模式下全部的功能一样？

### DISCUSSION THREAD

**Discussion by shenyan-008** [immich-app/immich#13447] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I can't batch select and edit when using "Folder" mode. Can you add all the functions to make it the same as in "Photo" mode?/使用“文件夹”模式时无法批量选择，编辑。能否添加全部的功能，使其与“照片”模式下全部的功能一样？

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Dupe of #9159
